pages/team.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: MDAnalysis team
+---
+
+MDAnalysis is a community-driven project that is made possible through the efforts of many members who contribute in numerous and diverse ways, ranging from direct package development, maintenance, documentation, communication, and managerial responsibilities. On this page we list identified project roles and those historically responsible for the tasks. In the future, we plan to also list the contributors currently responsible for overseeing and managing associated tasks. We note that the listed roles on this page can differ significantly in scope and required effort.
+
+## Roles
+
+In this section we list contributors who have led or contributed substantial effort to the roles listed below.
+At present, we have only listed leading figures from the past 2-3 years, i.e. from 2020 onwards.
+However, we hope to eventually acknowledge everyone who has been responsible for a particular role throughout
+the lifetime of the project.
+
+If you have participated to MDAnalysis in the past and feel your previous/historical role could be better characterised,
+please contact us at mdanalysis@numfocus.org to let us know!
+
+| Role                         | Sub-role                       | Historical                                                              |
```

Comment:
```suggestion
| Role                         | Sub-role                       | Current Team  (leads are bolded)                                                             | Historical                                                            |
```

Reply:
Suggested headers for reorganizing the table such that it includes current team members. I won't make these changes throughout since I don't want to mess with the Markdown table formatting until we have reached consensus on whether this is how we would like to organize everything.

Reply:
Changed!